tm2/pkg/crypto/keys/client/root.go

```diff
@@ -28,7 +28,6 @@ func NewRootCmdWithBaseConfig(io commands.IO, base BaseOptions) *commands.Comman
 			ShortUsage: "<subcommand> [flags] [<arg>...]",
 			LongHelp:   "Manages private keys for the node",
 			Options: []ff.Option{
-				ff.WithConfigFileFlag("config"),
```

Comment:
Why?

Reply:
because `--config` is replaced with `--home`, which is not a file